zng/docs/spec.md

```diff
@@ -6,12 +6,37 @@
 > zq attempts to implement everything herein excepting:
 >
 > * the bytes type is not yet implemented,
-> * ordering hints are not generated or taken advantage of,
```

Comment:
I've realized late in the PR review process that this was a dangling reference, as mention of ordering hints was removed from the spec in April, 2020 as part of #508. There's plenty of chatter on Slack around April 3, 2020 among the Dev team all agreeing that it was appropriate to remove ordering hints from the spec since it had never been implemented and there were apparently no plans to implement it, so it seems the dangling reference should go as well.